cmd/root.go

```diff
@@ -59,26 +67,49 @@ var RootCmd = &cobra.Command{
 	Run: func(cmd *cobra.Command, args []string) {
 		log.Info("poke-ssl-agent starting")
 
-		schedulerEvents := core.NewConsumer()
+		quit := make(chan os.Signal, 2)
+		signal.Notify(quit, syscall.SIGTERM)
+		signal.Notify(quit, syscall.SIGINT)
+
+		consumer, err := core.NewConsumer()
+		if err != nil {
+			log.WithError(err).Fatal("Could not start the kafka consumer")
+		}
 
-		//quit := make(chan os.Signal)
-		//signal.Notify(quit, os.Interrupt)
+		events := make(chan *models.SchedulerEvent)
+		go func() {
+			for {
+				select {
+				case m := <-consumer.Messages():
```

Comment:
Check that the chan is open
```suggestion
				case m, ok := <-consumer.Messages():
```